ArduPlane/servos.cpp

```diff
@@ -905,13 +905,27 @@ void Plane::set_servos(void)
         // after an auto land and auto disarm, set the servos to be neutral just
         // in case we're upside down or some crazy angle and straining the servos.
         if (landing.get_then_servos_neutral() == 1) {
-            SRV_Channels::set_output_scaled(SRV_Channel::k_aileron, 0);
-            SRV_Channels::set_output_scaled(SRV_Channel::k_elevator, 0);
-            SRV_Channels::set_output_scaled(SRV_Channel::k_rudder, 0);
+            for(uint8_t i = 0; i < NUM_SERVO_CHANNELS ; i++) {
+                SRV_Channel *chan = SRV_Channels::srv_channel(i);
+                if(chan == nullptr){
+                    continue;
+                }
+
+                if(SRV_Channel::is_control_surface(chan->get_function())) {
+                    SRV_Channels::set_output_scaled(chan->get_function(), 0);
+                }
```

Comment:
```suggestion
                if(chan == nullptr || !SRV_Channel::is_control_surface(chan->get_function())){
                    continue;
                }
                if (landing.get_then_servos_neutral() == 1) {
                    SRV_Channels::set_output_scaled(chan->get_function(), 0);
                } else if (landing.get_then_servos_neutral() == 2) {
                     SRV_Channels::set_output_limit(chan->get_function(), SRV_Channel::Limit::ZERO_PWM);
                }
```
If you move the param check inside the for loop you will only need one.

Reply:
Done! I committed the suggested changes.  Moved the piece of code to its own dedicated function. I couldn't figure out what to name this function. Please check if my name works or suggest a name for it.